src/python/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Mastho - 2020
+link_libraries(kdmp-parser)
+
+include_directories("C:\\Python36\\include")
+link_directories("C:\\Python36\\libs")
```

Comment:
i guess it also breaks on x64 as the path needs to be adapted

Reply:
https://cmake.org/cmake/help/v3.12/module/FindPython3.html#module:FindPython3 this looks like the right thing to use

Reply:
example: https://github.com/JonathanSalwan/Triton/blob/master/CMakeLists.txt#L45